packages/dds/merge-tree/src/test/mergeTreeOperationRunner.ts

```diff
@@ -66,7 +65,16 @@ export function doOverRange(
     range: IConfigRange,
     growthFunc: (input: number) => number,
     doAction: (current: number) => void) {
+    let lastCurrent = Number.NaN;
     for (let current = range.min; current <= range.max; current = growthFunc(current)) {
+        // let growth funcs be simple
+        // especially around 0 and 1
+        // if the value didn't change,
+        // just increment it
+        if (current === lastCurrent) {
+            current++;
+        }
+        lastCurrent = current;
```

Comment:
this is test code limited to this package. currently its only used for integers, basically shorthand for loops that's easier to define in configuration. it probably wouldn't work right for non-integer cases, but i also don't know an easy way to limit to integers, but since its test code, any usage of non-integers would fail whatever new test was using it.

Reply:
re: the question: js ++ operators seem to align with C++ semantics; postfix ++ returns the value before increment whereas prefix ++ returns the value post-increment.

![image](https://user-images.githubusercontent.com/5222607/189744963-9d9737a6-eba5-4426-9a94-fcacab8e5197.png)

Reply:
Oh right, totally brain-farted on that one 😅 thanks!